src/Microsoft.DocAsCode.Build.JavaScriptReference/BuildInputs/ApiParameter.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.JavaScriptReference
```

Comment:
ManagedReference's model is shared between build and metadata command, so it's in DataContract. I think this model can be specific for JavaScript Document.

---
In reply to: [93179522](https://github.com/dotnet/docfx/pull/1016#discussion_r93179522) [](ancestors = 93179522)